src/components/svgs/nodes/mrt-line-badge.tsx

```diff
@@ -0,0 +1,216 @@
+    const onPointerMove = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerMove(id, e),
+        [id, handlePointerMove]
+    );
+    const onPointerUp = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerUp(id, e),
+        [id, handlePointerUp]
+    );
+
+    const width = 22.85;
+    const height = 12.935;
+
+    const fgColor = color[3];
+    const bgColor = color[2];
+
+    return (
+        <g
+            id={id}
+            transform={`translate(${x}, ${y})`}
+            onPointerDown={onPointerDown}
+            onPointerMove={onPointerMove}
+            onPointerUp={onPointerUp}
+            style={{ cursor: 'move' }}
+        >
+            <rect
+                transform={`scale(1.8)`}
+                x={-width / 2}
+                y={-height / 2}
+                rx="3"
+                ry="6"
+                width={width}
+                height={height}
+                fill={bgColor}
+                stroke="white"
+                strokeWidth="1"
+            />
+            <text
+                className="rmp-name__mrt"
+                textAnchor="middle"
+                fill={fgColor}
+                fontSize="15"
+                dominantBaseline="central"
+                letterSpacing="-0.2"
+            >
+                {lineCode}
+            </text>
+            <text
+                className="rmp-name__mrt"
+                textAnchor="left"
+                x="28"
+                y={isTram ? 3.5 : -3}
+                fill="black"
+                fontSize="10"
+                letterSpacing="-0.2"
+            >
+                {name}
+            </text>
+            {!isTram &&
+                lines.split(',').map((n, i) => (
+                    <g key={i} transform={`translate(${34 + 14 * i}, ${4.5})`}>
+                        {n.trim() !== 'airport' ? (
+                            <>
+                                <circle r="6" fill={bgColor}></circle>
+                                <text
+                                    className="rmp-name__mrt"
+                                    textAnchor="middle"
+                                    fill={fgColor}
+                                    fontSize="9"
+                                    dominantBaseline="central"
+                                    letterSpacing="-0.2"
+                                >
+                                    {n}
+                                </text>
+                            </>
+                        ) : (
+                            <>
+                                <circle r="6" fill={bgColor}></circle>
+                                <image
+                                    width="12"
+                                    height="12"
+                                    x="-6"
+                                    y="-6"
+                                    href={import.meta.env.BASE_URL + `images/facilities/airport_sg.svg`}
```

Comment:
This might not work as expected in downloads. Have you tested it?

Reply:
I will remove it. 😅

Reply:
Change it to <path ...>